server/views/applications/view.njk

```diff
@@ -96,10 +96,10 @@
         {% if key.kind === 'api-client#web' %}
         <div class="govuk-summary-list__row">
           <dt class="govuk-summary-list__key">
-            Client Secret
+            <label for="{{ key.id+"-secret" }}">Client Secret</label>
           </dt>
-          <dd class="govuk-summary-list__value">
-            <span class="api-value" style="-webkit-text-security: disc;" id={{ key.id+"-secret" }}>{{ key.clientSecret }}</span>
+          <dd class="govuk-summary-list__value" style="padding-right: 0">
+            <input type="password" class="api-value" style="border:none; background-color:white; width: 100%; " id="{{ key.id+"-secret" }}" value="{{ key.clientSecret }}" disabled>
```

Comment:
Can we move inline css into our global css file?

Reply:
I could move the input styles but the list value one I don't think I could move, unless I make an id for one style?

Reply:
I don't think theres anything wrong with that.
I'd probably expect input to be a class or id as well or we might mess up the create and update forms.

Reply:
Added the styles to the css under an id and a class